.github/workflows/upgrade-flux.yaml

```diff
@@ -0,0 +1,83 @@
+name: Update Components
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 * * * *"
+  push:
+    branches: [main]
```

Comment:
Does this automatically run when someone pushes to `main`? I think we only want to run this on the `cron` job, once a day (if I am reading that correctly).

Reply:
you are right